graphistry/pygraphistry.py

```diff
@@ -2,6 +2,7 @@
 from typing_extensions import Literal
 from graphistry.Plottable import Plottable
 from graphistry.privacy import Mode, Privacy
+from IPython.display import DisplayObject, TextDisplayObject
```

Comment:
this shouldn't be a top-level import as it'd break many installs; note the dynamic import once we've sniffed